Templates/Kotlin/Common/Includes/ModelAPI.stencil

```diff
@@ -26,12 +26,12 @@ object {{ type }}API {
     {% endfor %}
     {% for schema in requestSchemas %}
 
-        {% filter indent:4 %}{% include "Common/Includes/Model.stencil" schema %}{% endfilter %}
+        {% filter indent:4 %}{% include "Common/Includes/ModelAPIBodyModel.stencil" schema %}{% endfilter %}
     {% endfor %}
 
     private val service: {{ type|upperCamelCase }}Service by lazy {
         Retrofit.Builder()
-            .client(OkHttpClient.Builder().addInterceptor(InterceptorUtils.getInterceptor({% if type == "GetTerms" %}"application/vnd.api+json", "application/vnd.api+json"{% else %}"application/json", "application/json"{% endif%})).build())
+            .client(OkHttpClient.Builder().addNetworkInterceptor(InterceptorUtils.getInterceptor("application/vnd.api+json", "application/vnd.api+json")).build())
```

Comment:
@Melle010 Will this break any currently used requests?

Reply:
Nope, tested all uses APIs.